tests/virtualenv_tools_test.py

```diff
@@ -61,6 +61,18 @@ def test_already_up_to_date(venv, capsys):
     assert out == 'Already up-to-date: {0} ({0})\n'.format(venv.before)
 
 
+def test_bourne_shell_exec(venv):
+    bindir = venv.before.join('bin')
+
+    bindir.join('bourne.py').write_binary(
+        bytes("""#!/bin/sh
+'''exec' {} "$0" "$@"
+' '''
+""".format(venv.before.strpath), 'utf-8'))
+
+    run(venv.before, venv.after)
```

Comment:
there should be some assertion about the output -- this test passes without your change

Reply:
Another place where I copied an example in a different context. I have added the assertion.